bin/servicerouter.py

```diff
@@ -437,7 +437,8 @@ def api_req_raw(self, method, path, body=None, **kwargs):
             )
             if response.status_code == 200:
                 break
-
+        if 'message' in response.json():
+            response.reason = "%s (%s)" % (response.reason, response.json()['message'])
```

Comment:
This line is longer than 79 characters, needs to be wrapped to be PEP-8 compliant. See: https://www.python.org/dev/peps/pep-0008/#maximum-line-length

I added a check for pep8 in #2045 + some docs so hopefully pep-8 violations will be easier to see / fix early next time.

Other than that, seems reasonable to me to do, happy to merge once it is pep-8 compliant.